sql-statements/sql-statement-table.md

```diff
@@ -0,0 +1,63 @@
+---
+title: TABLE | TiDB SQL Statement Reference
+summary: An overview of the usage of TABLE for the TiDB database.
+aliases: ['/docs/dev/sql-statements/sql-statement-table/']
+---
+
+# TABLE
+
+The `TABLE` statement can be used instead of `SELECT * FROM` when no aggregation or complex filtering is needed. 
+
+## Synopsis
+
+```ebnf+diagram
+TableStmt ::=
+    "TABLE" Table ( "ORDER BY" Column )? ( "LIMIT" NUM )?
+```
+
+## Examples
+
+```sql
+tidb 5.7.25-TiDB-v5.0.0 > CREATE TABLE t1(id INT PRIMARY KEY);
+Query OK, 0 rows affected (0.31 sec)
+
+tidb 5.7.25-TiDB-v5.0.0 > INSERT INTO t1 VALUES (1),(2),(3);
+Query OK, 3 rows affected (0.06 sec)
+Records: 3  Duplicates: 0  Warnings: 0
+
+tidb 5.7.25-TiDB-v5.0.0 > TABLE t1;
++----+
+| id |
++----+
+|  1 |
+|  2 |
+|  3 |
++----+
+3 rows in set (0.01 sec)
+
+tidb 5.7.25-TiDB-v5.0.0 > TABLE t1 ORDER BY id DESC;
++----+
+| id |
++----+
+|  3 |
+|  2 |
+|  1 |
++----+
+3 rows in set (0.01 sec)
+
+tidb 5.7.25-TiDB-v5.0.0 > TABLE t1 LIMIT 1;
++----+
+| id |
++----+
+|  1 |
++----+
+1 row in set (0.01 sec)
+```
```

Comment:
Our new code standard is to include the SQL as copyable, then the output. i.e. (ignore the \ character - I can't seem to get the suggestion working):

```suggestion

{{< copyable "sql" >}}

\```sql
CREATE TABLE t1(id INT PRIMARY KEY);
INSERT INTO t1 VALUES (1),(2),(3);
TABLE t1;
TABLE t1 ORDER BY id DESC;
TABLE t1 LIMIT 1;
SELECT * FROM t1;
\```

\```sql
Query OK, 0 rows affected (0.31 sec)

Query OK, 3 rows affected (0.06 sec)
Records: 3  Duplicates: 0  Warnings: 0

+----+
| id |
+----+
|  1 |
|  2 |
|  3 |
+----+
3 rows in set (0.01 sec)

+----+
| id |
+----+
|  3 |
|  2 |
|  1 |
+----+
3 rows in set (0.01 sec)

+----+
| id |
+----+
|  1 |
+----+
1 row in set (0.01 sec)

+----+
| id |
+----+
|  1 |
|  2 |
|  3 |
+----+
3 rows in set (0.01 sec)

\```
```